src/pint/models/astrometry.py

```diff
@@ -69,6 +63,22 @@ def ssb_to_psb_xyz_ECL(self, epoch=None):
         # TODO: would it be better for this to return a 6-vector (pos, vel)?
         return self.coords_as_ECL(epoch=epoch).cartesian.xyz.transpose()
 
+    def sun_angle(self, toas):
+        """Compute the pulsar-Earth-Sun angle.
+
+        This angle takes into account the motion of the Sun around the solar system barycenter.
+        """
+        tbl = toas.table
+
+        obs_vec = tbl["ssb_obs_pos"].quantity
+        # FIXME: sun pos may not have been computed
+        sun_vec = tbl["obs_sun_pos"].quantity
+        psr_vec = self._parent.ssb_to_psb_xyz_ICRS(epoch=tbl["tdb"])
```

Comment:
`ssb_to_psb_xyz_ICRS` is in the class. Any reason to use it from the `._parent`?